### PR TITLE
Add first version of Gibbs notebook

### LINKED ISSUE #15: Add Gibbs notebook to training material

### PR DESCRIPTION
**Pull request by nabriis**
Closes #15

### CODE REVIEW

File: training/ExerciseXX_Gibbs.ipynb [hunk 1]
```diff
@@ -0,0 +1,696 @@
...
+        "Going through this notebook you will learn to:\n",
+        "\n",
+        "- Describe the basic idea of Gibbs sampling.\n",
+        "- Define a hierarchical Bayesian model using CUQIpy.\n",
+        "- Define a Gibbs sampling scheme in CUQIpy.\n",
+        "- Run the Gibbs sampler and analyze the results.\n",
+        "\n",
+        "## Table of contents\n",
+        "\n",
+        "TODO\n"
```
Comment (jakobsj) [inline training/ExerciseXX_Gibbs.ipynb:30]: Add table of contents

File: training/ExerciseXX_Gibbs.ipynb [hunk 2]
```diff
@@ -0,0 +1,696 @@
+{
```
Comment (jakobsj) [inline training/ExerciseXX_Gibbs.ipynb:1]: Replace XX in filename by 06.

**Review by jakobsj — APPROVED** [review]
See minor points in line comments.

Will add PR into this in a sec.

Notebook nicely explains Gibbs. Not yet as polished as other notebooks but easily good enough to merge as a first version with the above fixed.

**Comment by nabriis** [comment]
I will look into it tomorrow morning.

**Comment by jakobsj** [comment]
Wow, yes I see.  Wasn't much. I think one was for the setting of the path to use ../../CUQIpy and maybe there was written cuqi or cuqipy somewhere that should be CUQIpy. Perhaps some typos also that we can live with.

File: training/ExerciseXX_Gibbs.ipynb (hunk 1 above)
Comment (nabriis) [reply training/ExerciseXX_Gibbs.ipynb]: Done

File: training/ExerciseXX_Gibbs.ipynb (hunk 2 above)
Comment (nabriis) [reply training/ExerciseXX_Gibbs.ipynb]: Done

**Comment by nabriis** [comment]
I ended with a final section comparing to BayesianProblem (2 lines of code).

**Pull request merged**